example/extended_ml_workflow.py

```diff
@@ -0,0 +1,130 @@
+"""
+An extended workflow of the toy ML workflow as defined in `example.ml_workflow`.
+The extended example can also return the ROC AUC score and allows users to switch
+to a larger UCI ML covertype dataset, which can be found at:
+http://archive.ics.uci.edu/ml/datasets/Covertype
+"""
+
+import pandas as pd
+from sklearn import datasets, metrics
+from example.ml_workflow import basic_flow
+import time
+
+import bionic as bn
+
+# Convert the example ml workflow to a builder we'll use for extending.
+builder = basic_flow.to_builder()
+
+
+# Predict probabilities for the test data.
+@builder
+def prediction_frame(model, test_frame):
+    """
+    A dataframe with probability predictions for the test data.
```

Comment:
We should mention the name of the predictions column -- that's important for anyone who wants to consume this.

Maybe: "A dataframe with one column, ``proba``, containing predicted probabilities for the test data."